runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -203,6 +203,22 @@ typedef struct nnfw_train_info
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 } nnfw_train_info;
 
+constexpr char traininfo_metadata_name[] = "CIRCLE_TRAINING";
+
```

Comment:
This constexpr is intended to implement a circle+ loader. It is not for the users. Then how about moving this expression into the source file?

Reply:
Thanks for pointing out.
Updated!